### ISSUE TITLE
Can't use v3 in go module. Keeps begging for flimzy/kivik

### ISSUE THREAD

**Issue by vpmv** [go-kivik/kivik#525]
1. Which version of Kivik are you using? Please state a version number, or `master`.
I'm trying to use v3, but it keeps going back to `github.com/go-kivik/kivik v2.0.0+incompatible`

2. Which version of Go are you using? (Output of `go version`) -- Kivik 2.x supports Go 1.9 and later. Kivik 3.x requires Go 1.11 or later, with module support. The Kivik 4/master aims to support Go 1.11 and later, with module support.

Go 1.14.7

3. What did you do? (Include your code if possible)

Modules in .go import:
```
"github.com/flimzy/kivik/v3"
_ "github.com/go-kivik/couchdb/v3"
```

Go.mod file:
```
github.com/go-kivik/couchdb/v3 v3.2.5
github.com/go-kivik/kivik v3.2.3
```

I tried adding `flimzy/kivik v3.2.3` and `flimzy/kivik/v3`, but they both give different erroneous responses.

4. What did you expect to see?
Valid imports

5. What did you see instead?

when adding flimzy/kivik/v3:
```
go: github.com/flimzy/kivik/v3@v3.2.3: parsing go.mod:
	module declares its path as: github.com/go-kivik/kivik/v3
	        but was required as: github.com/flimzy/kivik/v3
```
without /v3
```
require github.com/flimzy/kivik: version "v3.2.3" invalid: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v3
```
Omitting flimzy fails to import `go-kivik/kivik` entirely.
Omitting /v3 after the modules will import v2

This is the first time I encouter this kind of problem. Pls help.

**Comment by flimzy** [comment]
```golang
"github.com/flimzy/kivik/v3"
```
should be

```golang
"github.com/go-kivik/kivik/v3"
```

**Comment by vpmv** [comment]
Thanks for the quick reply. 
Sadly, that is my exact problem.

github.com/go-kivik/couchdb/v3 v3.2.5 -- imports fine
github.com/go-kivik/kivik/v3 v3.2.3 -- stays grayed out, not recognized in imports

**Comment by flimzy** [comment]
`github.com/flimzy/kivik` should not be used anywhere. Search your entire code base for that import path, and replace it with `github.com/go-kivik/kivik`. That's what's causeing the error message you see.

**Comment by vpmv** [comment]
Oh, my. I've been trying so many solutions and this is a byproduct of one. Let's blame the early morning and lack of coffee :) 

TLDR: My bad. 
Thanks for the support!
